src/test/java/plugins/LdapPluginTest.java

```diff
@@ -205,7 +249,7 @@ public void login_use_fallback_server() {
         int freePort = this.findAvailablePort();
         LdapDetails ldapDetails = new LdapDetails("", 0, ldapContainer.getManagerDn(), ldapContainer.getManagerPassword(), ldapContainer.getRootDn());
         // Fallback-Config: primary server is not running, alternative server is running docker fixture
-        ldapDetails.setHostWithPort("localhost:" + freePort + " localhost:" + ldapContainer.getPort());
+        ldapDetails.setHostWithPort(ldapContainer.getHost()+":" + freePort + " " + ldapContainer.getHost() + ":" + ldapContainer.getPort());
```

Comment:
> Just want to point out that this issue will still be present for localhost and linux.

That's not an issue are `findAvailableFreePort()` finds a port that there is nothing listening on - and as I stated there is guaranteed to not be anything listeing on this.  Userlandproxy which is used by docker to do the rewrites IIUC needs to bind a port.  So unless you have a firewall rewriting rule for some random ports (at which point I would balk at your setup) this is safe on linux with docker running on localhost.